### PR TITLE
VACMS-15647 Add new vamc mental health field.

### PR DESCRIPTION
**Pull request by swirtSJW**
## Description

Relates to #15647

## QA steps

What needs to be checked to prove this works?
What needs to be checked to prove it didn't break any related things?
What variations of circumstances (users, actions, values) need to be checked?

As `QA Content Editor`
1. Go to [edit a VAMC facility](https://pr15679-nx274uz2uav24soqj9f879jn7bgfsbp6.ci.cms.va.gov/node/2226/edit) 
   - [x] validate that there is the new field "Mental health contact phone number" sitting beneath the Facility API data section and is editable.
   - [x] Validate the field `Mental health phone number` is in the  locked api section and is NOT editable.
2. Then enter a phone number for "Mental health contact phone number" and save the change as published.
   - [x] Validate that Mental health contact phone number shows up after save.
3. odds and ends
   - [x] Validate the label for the locked "Phone number" has only P capitalized
   - [x] Validate "Mental health phone number"  no longer says "Mental Health Phone"

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.
- [ ] If there are field changes, front end output has been thoroughly checked.

### Select Team for PR review

- [ ] `CMS Team`
- [ ] `Public websites`
- [ ] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping the UX writer so they are ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping the UX writer to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

**Review by omahane — CHANGES_REQUESTED** [review]
This looks great. 
Just one change to the instructional text.

File: config/sync/field.field.node.health_care_local_facility.field_mental_health_contact_phon.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    - tmgmt_content
+third_party_settings:
+  tmgmt_content:
+    excluded: false
+id: node.health_care_local_facility.field_mental_health_contact_phon
+field_name: field_mental_health_contact_phon
+entity_type: node
+bundle: health_care_local_facility
+label: 'Mental health contact phone number'
+description: 'Enter the number Veterans should call to access mental health services at your facility. If your facility doesn''''t offer these services, enter the most appropriate system number. This number will be displayed everywhere your facility appears across VA.gov. <a href="/help/va-medical-centers-vamcs/about-locations-content-for-vamcs/how-to-edit-a-vamc-facility"  target="_blank">View guidelines for editing VAMC facilities (opens in a new tab)</a>'
```
[the remark this answers is not in the document]
Comment (swirtSJW) [reply config/sync/field.field.node.health_care_local_facility.field_mental_health_contact_phon.yml:19]: Good catch.

[the remark this answers is not in the document]
Comment (swirtSJW) [reply config/sync/field.field.node.health_care_local_facility.field_mental_health_contact_phon.yml:19]: Oof but your fix only removes one, it has to remove 2.  I can take care of that.

[the remark this answers is not in the document]
Comment (swirtSJW) [reply config/sync/field.field.node.health_care_local_facility.field_mental_health_contact_phon.yml]: Fixed 
image

[the remark this answers is not in the document]
Comment (omahane) [reply config/sync/field.field.node.health_care_local_facility.field_mental_health_contact_phon.yml]: Oh, dang. Good catch of my catch.

**Pull request merged**